modules/persistent-storage-csi-smb-cifs-create-pv.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-pv_{context}"]
+= Creating PVs/PVCs with SMB shares
+
+To create a persistent volume (PV) and persistent volume claim (PVC) to consume Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+* Storage class for CIFS/SMB already created (see _Creating a storage class for CIFS/SMB_).
+
+.Procedure
+
+. Create a PV:
++
+Run the following command with the following example YAML file:
```

Comment:
I personally feel it reads a bit weird to have two sentences like this ending with a colon in one step, and also since the command has a user replaced value, I feel like you don't have to mention "example". I'd suggest the following rewording:
```suggestion
. Create a PV by running the following command:
```
I'm not sure whether this maintains the technical accuracy of telling users to use the sample information below, though.